.changeset/slimy-clouds-roll.md

```diff
@@ -0,0 +1,7 @@
+---
+'lit-html': patch
+'lit': patch
+'lit-element': patch
```

Comment:
This isn't touching anything in `lit-element`. I think it's okay not to bump that but keep `lit`?
```suggestion
'lit-html': patch
'lit': patch
```

Reply:
I think we need to revisit our convention here and write down the decision and rationale, so it's easy to recall.